HelpSource/Reference/Assignment.schelp

```diff
@@ -14,6 +14,50 @@ x = [1, 2, 3, 4].rotate(1);
 c = a + b;
 ::
 
+section:: Assignment in Declaration
+
+When declaring a variable, a value can be assigned to it immediately:
+
+code::
+(
+var x = 1;
+var y = -1;
+x + y
+)
+::
+
+Such assignments can be chained up:
```

Comment:
Does this really need to be stated? It's already shown in the example above this section:

```supercollider
x = [1, 2, 3, 4].rotate(1);
c = a + b;
```

it's also unclear what is meant by "chained up" - IMO better to remove this section and/or combine the examples

Reply:
done.